src/Player.cpp

```diff
@@ -25,14 +25,14 @@ bool Player::loadData(const std::string & data){
 }
 
 void Player::setName() {
-	size_t frontLoop = 0;
+    size_t frontLoop = 0;
 
-	std::string temp = "";
-	while (data_.at(frontLoop) != ':') {
-		temp += data_.at(frontLoop);
+    std::string temp = "";
+    while (data_.at(frontLoop) != ':') {

```

Comment:
What about std::copy or std::find algorithms?

Reply:
@Tomkor9  it's your part of code, could you please take a look?

Reply:
It's now completely rebuilt and works correctly.